pkg/storage/watch/update/associated.go

```diff
@@ -0,0 +1,10 @@
+package update
+
+import (
+	"github.com/weaveworks/ignite/pkg/storage"
+)
+
+type AssociatedUpdate struct {
```

Comment:
godoc

Reply:
Added in https://github.com/weaveworks/ignite/pull/242.